docker/Dockerfile.sles.12.3

```diff
@@ -17,23 +17,29 @@ RUN useradd -u $UID -ms /bin/bash $USER
 RUN groupadd -g $UID $USER
 
 # Install basic tools
-RUN zypper --non-interactive update
+RUN zypper --non-interactive --no-gpg-checks refresh; \
+    zypper --non-interactive update
 
 # basic building
-RUN zypper --non-interactive install autoconf automake build                  \
+RUN zypper --non-interactive --no-gpg-checks refresh; \
+    zypper --non-interactive install autoconf automake build                  \
```

Comment:
You want this `zypper ... install ... build` to be done down after the `openSUSE:/Tools/SLE_12_SP3/openSUSE:Tools.repo` is added because the [build](https://download.opensuse.org/repositories/openSUSE:/Tools/openSUSE_12.3/noarch/build-20190709-318.1.noarch.rpm) that's in that repo is much newer than what's in stock SLES 12.3.

I wonder if using an old `build` is the cause of some of your issues.